Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs

```diff
@@ -646,10 +646,18 @@ internal static async Task WaitInteractiveModeAsync(this UITestContext context)
         manifest.Tags.Any(tag => tag.EqualsOrdinalIgnoreCase(value: ManifestConstants.AdminTag));
 
     private static async Task<bool> PermissionExistsAsync(
-        IEnumerable<IPermissionProvider> permissionProviders, string permissionName) =>
-        (await Task.WhenAll(permissionProviders.Select(provider => provider.GetPermissionsAsync())))
-            .SelectMany(permissions => permissions)
-            .Any(permission => permission.Name == permissionName);
+        IEnumerable<IPermissionProvider> permissionProviders, string permissionName)
```

Comment:
Please break down all function arguments.
```suggestion
        IEnumerable<IPermissionProvider> permissionProviders,
        string permissionName)
```